syft/formats/syftjson/encoder_test.go

```diff
@@ -61,7 +62,13 @@ func TestEncodeFullJSONDocument(t *testing.T) {
 		FoundBy:      "the-cataloger-1",
 		Language:     pkg.Python,
 		MetadataType: pkg.PythonPackageMetadataType,
-		Licenses:     []string{"MIT"},
+		Licenses: []pkg.License{
+			{
```

Comment:
nit: try to use the constructors you added instead of raw structs (e.g. `pkg.NewLicense("MIT")`)

Reply:
I think I found all cases where we were not using the constructor and added it (-337 lines 👍 )!